hibernate-core/src/main/java/org/hibernate/boot/archive/internal/ArchiveHelper.java

```diff
@@ -93,7 +93,7 @@ else if ( "zip".equals( protocol )
 					"Unable to determine JAR Url from " + url + ". Cause: " + e.getMessage()
 			);
 		}
-		log.trace( "JAR URL from URL Entry: " + url + " >> " + jarUrl );
```

Comment:
The above string concatenation will incur performance cost invariably even though trace logging level is disabled